Riot/Views/RoomList/RoomCollectionViewCell.m

```diff
@@ -123,16 +123,8 @@ - (void)render:(MXKCellData *)cellData
                 
```

Comment:
It is too early to update the `RoomCollectionViewCell` class.
Please remove all the changes for this class in this PR.

The display with a large text size is not relevant:
![screen shot 2017-10-30 at 19 40 15](https://user-images.githubusercontent.com/8969772/32189151-32439c6a-bdaa-11e7-9cbd-9a475fc3e4e3.png)